src/main/java/technology/semi/weaviate/client/v1/graphql/query/RawGraphQL.java

```diff
@@ -0,0 +1,28 @@
+package technology.semi.weaviate.client.v1.graphql.query;
+
+import technology.semi.weaviate.client.Config;
+import technology.semi.weaviate.client.base.BaseClient;
+import technology.semi.weaviate.client.base.ClientResult;
+import technology.semi.weaviate.client.base.Response;
+import technology.semi.weaviate.client.base.Result;
+import technology.semi.weaviate.client.v1.graphql.model.GraphQLQuery;
+import technology.semi.weaviate.client.v1.graphql.model.GraphQLResponse;
+
+
+
+public class RawGraphQL extends BaseClient<GraphQLResponse> implements ClientResult<GraphQLResponse> {
+  private  String query;
+ 
+  public RawGraphQL(Config config, String query) {
```

Comment:
Could you rename class to `Raw` to match remaining gql clients here (`Get`, `Explore`, `Aggregate`)